pkg/dd/monitor.go

```diff
@@ -0,0 +1,51 @@
+package dd
+
+import (
+	"fmt"
+	"strconv"
+
+	aw "github.com/deanishe/awgo"
+	"gopkg.in/zorkian/go-datadog-api.v2"
+)
+
+type Monitor struct {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [exported type Monitor should have comment or be unexported](https://app.codacy.com/manual/nekottyo/alfred-datadog-workflow/pullRequest?prid=5421329)